### PR TITLE
i18n(zh-cn): Translate `recipes/reading-time.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### What kind of changes does this PR include?

- Translated content

#### Description

Translate `recipes/reading-time.mdx` content to Chinese.

### CODE REVIEW

**Comment by 100gle** [comment]
cc @liruifengv

File: src/content/docs/zh-cn/recipes/reading-time.mdx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+       remarkPlugins: [remarkReadingTime],
+     },
+   });
+   ```
+
+   现在，所有的 Markdown 文档都会在其 frontmatter 中包含一个计算后的 `minutesRead` 属性。
+
+4. 展示阅读时间
+
+   如果你的博客文章存储在 [内容集合](/zh-cn/guides/content-collections/)中，那么也可以从 `entry.render()` 函数访问 `remarkPluginFrontmatter`，然后在模板中根据需要来决定渲染 `minutesRead` 属性的位置。
```
Comment (leohxj) [inline src/content/docs/zh-cn/recipes/reading-time.mdx:73]:
```suggestion
   如果你的博客文章存储在 [内容集合](/zh-cn/guides/content-collections/) 中，那么也可以从 `entry.render()` 函数访问 `remarkPluginFrontmatter`，然后在模板中根据需要来决定渲染 `minutesRead` 属性的位置。
```

File: src/content/docs/zh-cn/recipes/reading-time.mdx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+     <head>...</head>
+     <body>
+       ...
+       <p>{remarkPluginFrontmatter.minutesRead}</p>
+       ...
+     </body>
+   </html>
+   ```
+
+   如果您正在使用 [Markdown 布局](/zh-cn/guides/markdown-content/#markdown-和-mdx-页面)，那么在布局模板中也可以直接通过 `Astro.props` 来获取 frontmatter 中的 `minutesRead` 属性。
```
Comment (leohxj) [inline src/content/docs/zh-cn/recipes/reading-time.mdx:101]:
```suggestion
   如果你正在使用 [Markdown 布局](/zh-cn/guides/markdown-content/#markdown-和-mdx-页面)，那么在布局模板中也可以直接通过 `Astro.props` 来获取 frontmatter 中的 `minutesRead` 属性。
```

**Comment by leohxj** [comment]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work, peeps!

**Pull request merged**